src/main/resources/AkkaTracingAspect.template

```diff
@@ -28,12 +29,12 @@ class AkkaTracingAspect {
   private val config = ConfigFactory.load("<<<CONFIG>>>")
   private val configurationReader = new ConfigurationReader(config, classLoader)
 
-  private val collector = configurationReader.getCollector
-
   private val actorFilter = configurationReader.getActorFilter
   private val messageFilter = configurationReader.getMessageFilter
 
-  private implicit val formats = DefaultFormats
+  private implicit val formats: Formats = DefaultFormats ++ JavaTypesSerializers.all
+
+  private lazy val collector = ActorSystemUtils.getCollector(configurationReader)
```

Comment:
Could it work without `lazy`?

Reply:
I think that there was a problem with too early actor system initialization and that is why it is a lazy val.